rules/crowdstrike_rules/crowdstrike_dns_request.py

```diff
@@ -1,17 +1,20 @@
+from panther_base_helpers import get_crowdstrike_field
+
 # baddomain.com is present for testing purposes. Add domains you wish to be alerted on to this list
 DENYLIST = ["baddomain.com"]
 
 
 def rule(event):
-    if event.get("DomainName") in DENYLIST:
+    if get_crowdstrike_field(event, "DomainName") in DENYLIST:
```

Comment:
I think you'll also need to check that the event type is correct, when `event['p_log_type'] == 'Crowdstrike.FDREvent'`?

Reply:
To clarify on the above, we need to return early if this is the event type is not `DnsRequest` I believe.

Reply:
Fixed.